src/game/common/trect.h

```diff
@@ -19,7 +19,6 @@
 #define TRECT_H
 
 #include "always.h"
-#include "minmax.h"
```

Comment:
Needs to add `#include <algorithm>` here, in case it isn't included by things higher up the include chain.